src/lib.rs

```diff
@@ -44,12 +44,15 @@ pub use crate::file_filter::*;
 use log::{error, warn};
 use std::fs;
 use std::io::{BufReader, Cursor};
+use std::path::PathBuf;
 use std::{
     collections::{btree_map, hash_map},
     path::Path,
 };
 use walkdir::WalkDir;
 
+pub type ResultTuple = (PathBuf, PathBuf, CovResult);  
```

Comment:
Could you define this in src/defs.rs like other type definitions?

Reply:
pushed